examples/aio-sandbox/site_to_markdown.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+import base64
+from playwright.async_api import async_playwright
+from agent_sandbox import Sandbox
+
+async def site_to_markdown():
+    # Initialize sandbox client
+    c = Sandbox(base_url="http://localhost:8080")
+    home_dir = c.sandbox.get_context().home_dir
+
+    # Browser: Automation to download HTML
+    async with async_playwright() as p:
+        browser_info = c.browser.get_info().data
+        page = await (await p.chromium.connect_over_cdp(browser_info.cdp_url)).new_page()
+        await page.goto("https://example.com", wait_until="networkidle")
+        html = await page.content()
```

Comment:
The client is hardcoded to connect to `localhost:8080`. For better flexibility, consider making the sandbox address and port configurable via command-line arguments, similar to how the `site` URL is handled.